src/main.css

```diff
@@ -232,9 +234,9 @@ select {
 */
 
 button,
-[type='button'],
-[type='reset'],
-[type='submit'] {
```

Comment:
If we keep this global, we are obligated to chat to Emily about QA.
(maybe we don't need QA off the back)



BUT could avoid it I think

- Move new CSS to, and scoped section

```css
.v2-ds button {
```

And wrap data catalog in `<body class="v2-ds" `

or something

(or even better, scope it to MUI component, but not necessary)

Reply:
The new components are not using this css file. I assume it got generated from running the build script.

Reply:
ah